src/cpp/shared/game/game.cpp

```diff
@@ -20,6 +20,23 @@ int game::GetRandomNumTroops(int creat) {
 
 extern int gbNoCDRom;
 
+__int16 creatureTypesAndQuantities[NUM_FACTIONS][3][3] = {
```

Comment:
Needs more descriptive name. Also, heroes can never get tier 3 creatures, so part of this data is dead (i.e.: never read).

Reply:
Ok. Are you suggesting that this data be left out? I understand that the additional data may not be used, but I didn't feel comfortable getting rid of it.

Reply:
You're not getting rid of it. It's still in the original decompilation.

Reply:
It won't be at all formatted properly in the original decompilation, but I'll go ahead and omit the tier 3 data.